ansible/roles/destroy-api-deployment-dns-entry/tasks/main.yml

```diff
@@ -1,4 +1,7 @@
 
+- name: print workspace
+  debug: msg="workspace - {{ workspace }}"
```

Comment:
Is this still required?

Reply:
yes .. surely the name 'print workspace' is comment enough on these ;)?